src/cmd/dev.go

```diff
@@ -79,11 +89,31 @@ var devDeployCmd = &cobra.Command{
 	},
 }
 
+// isLocalBundle checks if the bundle source is a local bundle
+func isLocalBundle(src string) bool {
+	return helpers.IsDir(src) || strings.Contains(src, ".tar.zst")
+}
+
+// validateDevDeployFlags validates the flags for dev deploy
+func validateDevDeployFlags(isLocalBundle bool) error {
+	if !isLocalBundle {
+		//Throw error if trying to run with --flavor, --flavor-all, or --force-create flag with remote bundle
```

Comment:
Is this because we append the pkg root manifest to the refs in the bundle YAML once it's been bundled? (and we don't want to modify the remote bundle?)

Reply:
yeah, flavor is applied on create so would need to recreate the remote zarf parckages with the new flavor. I'm sure this can be handled, was just going to be more complicated.